docs/readme-sync/v4.0/deploy-as-a-microservice/030 - use-optimizely-agent/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Use Optimizely Agent"
+excerpt: ""
+slug: "use-optimizely-agent"
+hidden: false
+metadata: 
+  title: "How to use Optimizely Agent - Optimizely Full Stack"
+createdAt: "2020-02-21T17:44:28.054Z"
+updatedAt: "2020-04-08T21:26:30.308Z"
+---
+
+Optimizely Agent provides [APIs](https://library.optimizely.com/docs/api/agent/v1/index.html) that enable running feature flag rules, such as A/B tests and targeted flag deliveries. Agent provides equivalent functionality to all our SDKs. At its core is the [Optimizely Go SDK](doc:go-sdk). 
+
+### Running feature flag rules
+
+
+The Decide [endpoint](https://library.optimizely.com/docs/api/agent/v1/index.html#operation/decide) buckets a user into a feature flag variation (choosing between multiple enabled or one disabled variation) as part of a flag rule. Flag rules include A/B tests and targeted feature flag deliveries. To run a flag rule, use:
+
+`POST /v1/decide?keys={flagKey}`
+
+In the request `application/json` body, include the `userID` and any `decideOptions`. The full request looks like this:
```

Comment:
use "userId" for consistency. Instead of userID- curl below uses "userId"

Reply:
yeah, if you wouldn't mind updating the examples separately that'd be great (I authored these docs before there was a working PR to test the examples against, FYI).

thanks, that "userID" was the only instance I found in docs; replaced